src/main/java/uc/dei/mse/supportivecare/service/mapper/NoticeMapper.java

```diff
@@ -1,8 +1,11 @@
 package uc.dei.mse.supportivecare.service.mapper;
 
 import java.util.Set;
-import org.mapstruct.*;
-import uc.dei.mse.supportivecare.domain.*;
+import org.mapstruct.BeanMapping;
```

Comment:
Change import to import org.mapstruct.*;

This will avoid some IDE errors due to generated sources